src/nodes.coffee

```diff
@@ -1683,6 +1697,12 @@ exports.Range = class Range extends Base
     args   = ', arguments' if hasArgs(@from) or hasArgs(@to)
     [@makeCode "(function() {#{pre}\n#{idt}for (#{body})#{post}}).apply(this#{args ? ''})"]
 
+  astProperties: -> {
```

Comment:
`Range` is a Coffeescript-specific node type, but its AST representation seems pretty clear-cut